### ISSUE TITLE
Does not expand some valid input

### ISSUE THREAD

**Issue by graymatter00** [madskristensen/ZenCodingVS#21]
### Installed product versions
- Visual Studio: Community 2022 (64-bit) Version 17.0.5
- This extension: 1.2.20

### Description
Does not expand some valid input.

### Steps to recreate
Tpye `div>p` and press `tab`.

### Current behavior
Expands to: `div><p></p>`.  
That is, the `div` tag is **not** expanded but the `p` tag is expanded.

### Expected behavior
Should expand to:
```
<div>
    <p></p>
<div>
```

### Additional info
Typing `div>p*2` and pressing `tab` works as expected.  Infact, `div>p*1` also works expected.

**Comment by Anoromi** [comment]
Doesn't occur to me in razor files with the same Visual Studio and extension version.

https://user-images.githubusercontent.com/72703146/150516429-d6e5c1c6-a5fa-4f72-a651-57b8a680bd1f.mp4

**Comment by graymatter00** [comment]
@Anoromi   My project is ASP.Net/MVC based, using VB.  Please try it in a vbhtml file to see if you can reproduce the issue.

**Comment by Anoromi** [comment]
@graymatter00 Yes, the bug does occur. Only in vbhtml file, not cshtml nor razor.

**Comment by graymatter00** [comment]
@Anoromi Thanks for the confirmation and clarification. 👍

**Comment by Anoromi** [comment]
@graymatter00 The issue doesn't occur when you create a file and after that rename it to vbhtml. Not sure if it will help you though.
